deploy/postgres/ansible/README.md

```
@@ -0,0 +1,157 @@
+- [ ] No encryption requirements (private network)
+
+### Software Dependencies
+
+- [ ] Ansible installed
+- [ ] Python3-bcrypt package
+- [ ] Python3-psycopg2 package
+- [ ] SSH keys configured for target server
+
+## Workflow Overview
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)